util/Android/scrollable.ts

```diff
@@ -0,0 +1,120 @@
+import IView from '../../ui/view/view';
+import ListViewItem from '../../ui/listviewitem';
+import AndroidUnitConverter from '../../util/Android/unitconverter';
+import ListView from '../../ui/listview';
+import GridView from '../../ui/gridview';
+
+const NativeRecyclerView = requireClass('androidx.recyclerview.widget.RecyclerView');
+const NativeSwipeRefreshLayout = requireClass('androidx.swiperefreshlayout.widget.SwipeRefreshLayout');
+
+export default class Scrollable {
+  protected _android: any; /**TODO: Check out with Cenk */
+  onPullRefresh(): void {}
+  nativeInner: any;
+  nativeObject: any;
+  nativeDataAdapter: any;
```

Comment:
also be private

Reply:
It will be used on listview-gridview, therefore making it protected instead

Reply:
Due to it being needed to be used on self = this and android part, I cannot make it protected or private 😢